includes/admin/class-wc-payments-admin.php

```diff
@@ -476,9 +476,8 @@ public function enqueue_payments_scripts() {
 	private function get_frontend_feature_flags() {
 		return array_merge(
 			[
-				'paymentTimeline'         => self::version_compare( WC_ADMIN_VERSION_NUMBER, '1.4.0', '>=' ),
-				'customSearch'            => self::version_compare( WC_ADMIN_VERSION_NUMBER, '1.3.0', '>=' ),
-				'accountOverviewTaskList' => self::is_account_overview_task_list_enabled(),
```

Comment:
We should probably also remove the `private static function is_account_overview_task_list_enabled() {()` since it's no longer used.

Reply:
Thanks for pointing it out, I missed that! Done in 06ad3136